local-tests/setup/session-sigs/get-pkp-session-sigs.ts

```diff
@@ -0,0 +1,45 @@
+import { LitActionResource, LitPKPResource } from '@lit-protocol/auth-helpers';
+import { LitAbility, LitResourceAbilityRequest } from '@lit-protocol/types';
+import { log } from '@lit-protocol/misc';
+import { LitNetwork } from '@lit-protocol/constants';
+import { TinnyEnvironment } from '../tinny-environment';
+import { TinnyPerson } from '../tinny-person';
+
+export const getPkpSessionSigs = async (
+  devEnv: TinnyEnvironment,
+  alice: TinnyPerson,
+  resourceAbilityRequests?: LitResourceAbilityRequest[]
+) => {
+  if (devEnv.litNodeClient.config.litNetwork === LitNetwork.Manzano) {
+    console.warn(
+      'Manzano network detected. Adding capacityDelegationAuthSig to pkpSessionSigs'
+    );
+  }
+
+  // Use default resourceAbilityRequests if not provided
+  const _resourceAbilityRequests = resourceAbilityRequests || [
+    {
+      resource: new LitPKPResource('*'),
+      ability: LitAbility.PKPSigning,
+    },
+    {
+      resource: new LitActionResource('*'),
+      ability: LitAbility.LitActionExecution,
+    },
+  ];
+
+  const pkpSessionSigs = await devEnv.litNodeClient.getPkpSessionSigs({
```

Comment:
For the private key reservation code to avoid nonce issues, shouldn't. we be reserving `alice`'s key here, and then releasing it when the test is done?

Perhaps we need a `withPerson()` wrapper function that is responsible for reserving the key a person will use, then running some code that will use their key, then releasing the key when that code is done, rather than sprinkle imperative code throughout all of these functions